test/api/tutorials_test.rb

```diff
@@ -30,13 +31,48 @@ def assert_tutorial_model_response(response, expected)
 
   #1: Testing for successful operation
   # POST /api/tutorials
-  def test_tutorials_post
+  
+  def test_tutorials_post_by_tutor
+    campus = FactoryBot.create(:campus)
+    unit = FactoryBot.create(:unit)
+    tutor = unit.tutors.first
+
+    tutorial = {
+      unit_id: unit.id,
+      tutor_id: tutor.id,
+      campus_id: campus.id,
+      capacity: 10,
+      abbreviation: 'LA011',
+      meeting_location: 'LAB34',
+      meeting_day: 'Tuesday',
+      meeting_time: '18:00'
+    }
+
+    data_to_post = {
+      tutorial: tutorial,
+    }
     number_of_tutorials = Tutorial.all.length
 
+    # perform the post with the unit first tutor auth token
+    post_json '/api/tutorials', with_auth_token(data_to_post, tutor)
```

Comment:
Tutors should not be able to post - my guess is that this tutor is also a convenor.

Reply:
You need to create a dedicated tutor user and add them to the unit